src/model/user-course.model.ts

```diff
@@ -0,0 +1,36 @@
+import { Entity, Column, PrimaryGeneratedColumn, CreateDateColumn, UpdateDateColumn, DeleteDateColumn } from 'typeorm'
+
+import { UserCourseLevel } from 'devu-shared-modules'
+
+@Entity('user-course')
```

Comment:
This determines the table name so it should probably use snake case rather than dashes to match postgres convention. Also we've been pluralizing the table names thus far. We can change that, but if we want to match consistency then this should be changed.

```suggestion
@Entity('user_courses')
```

PS - don't forget to update the migrations if we change the table name here